apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_fileoutputdatasource.cpp

```diff
@@ -48,11 +48,11 @@ TEST_F(FileOutputDataSourceTest, setMemberVariablePositive)
 {
 	unsigned short compare_channels = 3;
 	unsigned int compare_sampleRate = 32000;
-	int compare_pcmFormat = 1;
+	int compare_pcmFormat = media::AUDIO_FORMAT_TYPE_S8;
 
 	dataSource->setChannels(compare_channels);
 	dataSource->setSampleRate(compare_sampleRate);
-	dataSource->setPcmFormat(compare_pcmFormat);
+	dataSource->setPcmFormat(media::AUDIO_FORMAT_TYPE_S8);
```

Comment:
Please check whether this needs to be "media::AUDIO_FORMAT_TYPE_S16_LE" or not.

Reply:
In my opinion, this function checks only that the "set value" is same as the "get value". So no problems.